view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -56,19 +58,120 @@ define(
                 return this;
             },
 
+            /**
+             * Format Price
+             * 
+             * @param {float} amount - Amount to be formatted
+             * @return {string}
+             */
+            getFormattedAmount: function (amount) {
+                return priceUtils.formatPrice(amount, quote.getPriceFormat());
+            },
+
+            /**
+             * Get Installment minimum
+             * this function respects info from: https://www.omise.co/installment-payment
+             *
+             * NOTE: in the future this function should return data from capabilities object.
+             *
+             * @param {string} id - Bank ID
+             * @return {integer}
+             */
+            getInstallmentMinimum: function (id) {
+                switch (id) {
+                    case 'kbank': return 500;
+                    case 'bbl': return 500;
+                    case 'bay': return 300;
+                    case 'first_choice': return 300;
+                    case 'ktc': return 300;
+                }
+            },
+
+            /**
+             * Get Installment monthly interest rate
+             *
+             * NOTE: in the future this function should return data from capabilities object.
+             * 
+             * @param {string} id - Bank id
+             * @return {float}
+             */
+            getInstallmentInterestRate: function (id) {
+                switch (id) {
+                    case 'kbank': return 0.0065;
+                    case 'bbl': return 0.008;
+                    case 'bay': return 0.008;
+                    case 'first_choice': return 0.013;
+                    case 'ktc': return 0.008;
+                }
+            },
+
+            /**
+             * Get VAT tax rate
+             *
+             * NOTE: in the future this function should return data from capabilities object.
```

Comment:
Really? Doesn't seem related to the capabilities of a merchant account? Rather the economic policy of a country. Should come from config?

Reply:
we need to have it from capabilities. if vat will change for some reason is only one click on backend. in poland vat has been changed few times for last years.

Reply:
These are not in the new capabilities API, and there is no plan to put them there AFAIK. VAT rate is not related to merchant account capabilities

Reply:
in my opinion it is.. vat in different countries is different.
and vat can be changed.
imagine Thailand is increasing vat to 8 % like poland did many times.. and we have to rebuilt all plugins? that is crazy! it needs to be in capabilities.

when they will be developed more and more stuff will be added, and it depends on us what we will want and push backend team to do.

Reply:
VAT removed, no problem anymore..